pack/Microsoft.NET.Sdk.Web/Microsoft.NET.Sdk.Web.csproj

```diff
@@ -18,11 +18,11 @@
     <PackageReference Include="Microsoft.NET.Sdk.Web.ProjectSystem" Version="$(PackageVersion)" />
     <PackageReference Include="Microsoft.NET.Sdk.Publish" Version="$(PackageVersion)" />
 
-    <Content Include="$(WebSdkSource)\Web\Microsoft.NET.Sdk.Web.Targets\Sdk.props">
+    <Content Include="$(WebSdkSource)\Web\Microsoft.NET.Sdk.Web.Targets\*.props">
```

Comment:
Will need changes here also - https://github.com/aspnet/websdk/blob/master/build/test.proj#L42

We copy these targets to the local cli install folder so that we can run the end to end tests against the new bits.